static/js/recent_topics_data.js

```diff
@@ -6,16 +6,18 @@ export const topics = new Map(); // Key is stream-id:topic.
 export function process_message(msg) {
     // This function returns if topic_data
     // has changed or not.
-    if (msg.type !== "stream") {
-        // We don't process private messages yet.
+    if (msg.type !== "stream" && msg.type !== "private") {
         return false;
     }
-    // Initialize topic data
-    const key = get_topic_key(msg.stream_id, msg.topic);
+    // Initialize topic and pm data
+    // Key for private message is the user id's
+    // to whom the message is begin sent.
+    const key = msg.type === "stream" ? get_topic_key(msg.stream_id, msg.topic) : msg.to_user_ids;
```

Comment:
We should extract this to a function `get_key_from_message`.

Reply:
@amanagr is it okay to set the `pm_key` as `to_user_ids` ?

Reply:
It is fine but `recipient_id` is better. So, we should use `recipient_id` here, thanks for asking.